### PR TITLE
Add webhook dir

### PR DESCRIPTION
**Pull request by hexylena**
Honestly easier than PRing the galaxy codebase.

### CODE REVIEW

**Comment by gmauro** [comment]
Do not merge this. I am adding everything into the new_20.05_config branch

**Comment by hexylena** [comment]
I mean, this should be quite safe, should not affect anything you've changed in the new PR, and will merge cleanly with that branch. Do I need to retarget the 20.05 branch?

**Comment by hexylena** [comment]
I'm happy to update accordingly for the 20.05 branch if you like. I can undo the other webhook things in that branch (i.e. [`0dfe796` (#197)](https://github.com/usegalaxy-eu/infrastructure-playbook/pull/197/commits/0dfe796034e2e5cf1fa1ef50da356d4306c6f1c0) which is no longer needed.)

File: templates/nginx/galaxy-main.j2 [hunk 1]
```diff
@@ -236,6 +236,10 @@ server {
 		include uwsgi_params;
 	}
 
+	location /training-material/ {
```
Comment (bgruening) [inline templates/nginx/galaxy-main.j2:239]: can we add a comment here, that this is needed for the new shiny webhook?

**Comment by bgruening** [comment]
Thanks @hexylena this was super quick!

**Pull request merged**